### PR TITLE
Add project for improved Rakudo Star support

### PR DESCRIPTION
**Pull request by Tyil**

### CODE REVIEW

File: raku/rakudo-star-support.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Automated Rakudo Star Builds
+
+## Description
+
+Rakudo Star is currently build by hand on every release. This, in conjuction
+with a maintainer that doesn't use every OS in existence, causes targets other
+than GNU+Linux to be left behind. By leveraging automated builds, whether it be
+through GitHub, GitLab, SourceHut or anything else, support for other systems
+can be improved and kept up to date much easier.
+
```
Comment (JJ) [inline raku/rakudo-star-support.md:10]: Please add here a few milestones the student would be expected to reach; one or two will be enough.

File: raku/rakudo-star-support.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+## Expected Outcomes
+
+- On a push to master, automatically build a distribution tarball for:
+  - FreeBSD
+  - GNU+Linux (both glibc and musl)
+  - MacOS
+  - OpenBSD
+  - Windows
```
Comment (JJ) [inline raku/rakudo-star-support.md:19]: NetBSD? Is that still a thing?

File: raku/rakudo-star-support.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+## Required Skills
+
+- Bash
+- Git
+- GitHub Actions (or equivalent skills on any other platform)
+- Familiarity with build tooling of the required platforms
+
```
Comment (JJ) [inline raku/rakudo-star-support.md:30]: You can also use perl in GitHub actions :-) So this would go _both_ ways.

**Review by JJ — CHANGES_REQUESTED** [review]
Please clarify a few things and we're good.

**Comment by Tyil** [comment]
On Mon, Feb 01, 2021 at 08:52:36AM -0800, Juan Julián Merelo Guervós wrote:
> Please add here a few milestones the student would be expected to reach; one or two will be enough.

I was thinking of making every platform a single milestone, but I'm not sure if
that's what you're thinking of too. If that works for you, I can clarify this
in the post if desired.

-- 
With kind regards,

Patrick Spek


www:  https://www.tyil.nl/
mail: p.spek@tyil.nl
pgp:  1660 F6A2 DFA7 5347 322A  4DC0 7A6A C285 E2D9 8827

social: https://soc.fglt.nl/tyil
git:    https://home.tyil.nl/git

**Comment by Tyil** [comment]
On Mon, Feb 01, 2021 at 08:53:50AM -0800, Juan Julián Merelo Guervós wrote:
> Please clarify a few things and we're good.
> 
> [...]
> 
> NetBSD? Is that still a thing?

Yes it is, I'll include it in the list.

> You can also use perl in GitHub actions :-) So this would go _both_ ways.

Good to know!

-- 
With kind regards,

Patrick Spek


www:  https://www.tyil.nl/
mail: p.spek@tyil.nl
pgp:  1660 F6A2 DFA7 5347 322A  4DC0 7A6A C285 E2D9 8827

social: https://soc.fglt.nl/tyil
git:    https://home.tyil.nl/git

**Pull request merged**